modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-4_{context}"]
+= OADP 1.1.4 release notes
+
+The OADP 1.1.4 release notes lists any new features, resolve issues and bugs, and known issues.
+
+[id="new-features1.1.4_{context}"]
+== New features
+
+This version of OADP is a service release. No new features are added to this version.
+
+[id="resolved-issues1.1.4_{context}"]
+== Fixed bugs
+
+The following bugs have been fixed in this release:
+
+* link:https://issues.redhat.com/browse/OADP-1557[OADP-1557]
+* link:https://issues.redhat.com/browse/OADP-1822[OADP-1822]
+* link:https://issues.redhat.com/browse/OADP-1511[OADP-1511]
+* link:https://issues.redhat.com/browse/OADP-1642[OADP-1642]
+* link:https://issues.redhat.com/browse/OADP-1398[OADP-1398]
+* link:https://issues.redhat.com/browse/OADP-1789[OADP-1789]
+* link:https://issues.redhat.com/browse/OADP-1267[OADP-1267]
+* link:https://issues.redhat.com/browse/OADP-1389[OADP-1389]
+* link:https://issues.redhat.com/browse/OADP-1390[OADP-1390]
+* link:https://issues.redhat.com/browse/OADP-1650[OADP-1650]
+* link:https://issues.redhat.com/browse/OADP-1487[OADP-1487]
+* link:https://issues.redhat.com/browse/OADP-1668[OADP-1668]
+* link:https://issues.redhat.com/browse/OADP-1503[OADP-1503]
```

Comment:
Please remove these as it marked as not a bug/cannot reproduce.
https://issues.redhat.com/browse/OADP-1503
https://issues.redhat.com/browse/OADP-1789
https://issues.redhat.com/browse/OADP-1389

Reply:
@PrasadJoshi12 - you happy now? thanks very much

Reply:
Thank you !